### PR TITLE
Update build-arm64-firmware.md docs

### PR DESCRIPTION
**Pull request by dvescovi1**
add attr to build tools.
Remove $ so copy/paste is easier
Remove ~
Make commands "path relative"

### CODE REVIEW

**Comment by dvescovi1** [comment]
see issue #72 for more info

**Comment by dmcilvaney** [comment]
The new case sensitivity support looks neat: https://devblogs.microsoft.com/commandline/improved-per-directory-case-sensitivity-support-in-wsl/
This didn't exist when we first set it up, might be worth investigating for the 32 bit builds as well.

File: Documentation/build-arm64-firmware.md [hunk 1]
```diff
@@ -15,52 +15,52 @@ Note: The UEFI build environment has changed for 1903 and any existing build env
 
 1) Update and install build tools
     ```bash
-    $ sudo apt-get update
-    $ sudo apt-get upgrade
-    $ sudo apt-get install build-essential python python-dev python-crypto python-wand device-tree-compiler bison flex swig iasl uuid-dev wget git bc libssl-dev zlib1g-dev python3-pip
+    sudo apt-get update
+    sudo apt-get upgrade
+    sudo apt-get attr install build-essential python python-dev python-crypto python-wand device-tree-compiler bison flex swig iasl uuid-dev wget git bc libssl-dev zlib1g-dev python3-pip
```
Comment (neilsh-msft) [inline Documentation/build-arm64-firmware.md]: install needs to come first on apt-get. Change to apt-get install attr build-essential ...

File: Documentation/build-arm64-firmware.md [hunk 2]
```diff
@@ -15,52 +15,52 @@ Note: The UEFI build environment has changed for 1903 and any existing build env
...
     *** new for 1903 UEFI
-    $ sudo apt-get install gcc g++ make python3 mono-devel
+    sudo apt-get install gcc g++ make python3 mono-devel
     ***
-    $ pushd ~
-    $ wget https://releases.linaro.org/components/toolchain/binaries/7.2-2017.11/aarch64-linux-gnu/gcc-linaro-7.2.1-2017.11-x86_64_aarch64-linux-gnu.tar.xz
-    $ tar xf gcc-linaro-7.2.1-2017.11-x86_64_aarch64-linux-gnu.tar.xz
-    $ rm gcc-linaro-7.2.1-2017.11-x86_64_aarch64-linux-gnu.tar.xz
-    $ popd
+    pushd ~
```
Comment (neilsh-msft) [inline Documentation/build-arm64-firmware.md:24]: If you want to get rid of '~', then consider setting an environment variable with the current directory to use.  Ie:  export WORKSPACE=\`pwd\`

**Review by neilsh-msft — CHANGES_REQUESTED** [review]

File: Documentation/build-arm64-firmware.md (hunk 1 above)
Comment (dvescovi1) [reply Documentation/build-arm64-firmware.md]: opps

**Review by christopherco — APPROVED** [review]

File: Documentation/build-arm64-firmware.md (hunk 2 above)
Comment (dvescovi1) [reply Documentation/build-arm64-firmware.md:24]: I still like hosting the compiler in my home directory (but that's just me) so I would not totally get rid of ~ everywhere. If build script gets any more complicated I agree environment variable would be the next logical step.

**Pull request merged**